ocp_compatibility test broken due to expected output formatting

### Bug Summary

when running `ansible-test units --docker --python 3.11 --color` results in:
```
=========================== short test summary info ============================
FAILED tests/unit/filter/test_ocp_compatibility.py::TestOcpCompatibility::test_filter_with_deprecated_api_in_non_empty_list - AssertionError: {'4.1[100 chars]ault:eventtest-operator-service-account), p...
=================== 1 failed, 1 passed, 2 warnings in 0.19s ====================
FATAL: Command "pytest -r a -n auto --color yes -p no:cacheprovider -c /root/ansible/test/lib/ansible_test/_data/pytest/config/default.ini --junit-xml /root/ansible_collections/redhatci/ocp/tests/output/junit/python3.11-controller-units.xml --strict-markers --rootdir /root/ansible_collections/redhatci/ocp --confcutdir /root/ansible_collections/redhatci/ocp -v tests/unit/filter/test_ocp_compatibility.py" returned exit status 1.
Run command with stdout: podman exec --user pytest -i ansible-test-controller-hv4JgyIH sh -c 'tar cf - -C /root/ansible_collections/redhatci/ocp/tests --exclude .tmp output | gzip'
Run command with stdin: tar oxzf - -C /Users/makovgan/src/github.com/workspace/collections/ansible_collections/redhatci/ocp/tests
FATAL: Command "podman exec --user pytest ansible-test-controller-hv4JgyIH /usr/bin/env ANSIBLE_TEST_CONTENT_ROOT=/root/ansible_collections/redhatci/ocp LC_ALL=en_US.UTF-8 /usr/bin/python3.11 /root/ansible/bin/ansible-test units --containers '{}' --requirements-mode skip --verbose --truncate 230 --color yes --host-path tests/output/.tmp/host-2vpi9_nf --metadata tests/output/.tmp/metadata-x_xackob.json" returned exit status 1.
Run command: podman stop --time 0 ansible-test-controller-hv4JgyIH
Run command: podman rm ansible-test-controller-hv4JgyIH
```


### Details

expected the test to pass.
the fix is reformatting output text.

--- a/tests/unit/filter/test_ocp_compatibility.py
+++ b/tests/unit/filter/test_ocp_compatibility.py
@@ -53,19 +53,18 @@ class TestOcpCompatibility(unittest.TestCase):
 
             expected_result = {
                 "4.11": "compatible",
-                "4.12": " ".join([
-                    "events.v1beta1.events.k8s.io (service accounts: system:serviceaccount:default:",
-                    "eventtest-operator-service-account), podsecuritypolicies.v1beta1.policy ",
-                    "(service accounts: system:kube-controller-manager)"
+                "4.12": ", ".join([
+                    "events.v1beta1.events.k8s.io (service accounts: system:serviceaccount:default:eventtest-operator-service-account)",
+                    "podsecuritypolicies.v1beta1.policy (service accounts: system:kube-controller-manager)"
                 ]),
-                "4.13": " ".join([
-                    "events.v1beta1.events.k8s.io (service accounts: system:serviceaccount:default:",
-                    "eventtest-operator-service-account), podsecuritypolicies.v1beta1.policy ",
-                    "(service accounts: system:kube-controller-manager), flowschemas.v1beta1.",
-                    "flowcontrol.apiserver.k8s.io (service accounts: system:serviceaccount:",
-                    "openshift-cluster-version:default), prioritylevelconfigurations.v1beta1.",
-                    "flowcontrol.apiserver.k8s.io (service accounts: system:serviceaccount:",
-                    "openshift-cluster-version:default)"
+                "4.13": ", ".join([
+                    "events.v1beta1.events.k8s.io (service accounts: system:serviceaccount:default:eventtest-operator-service-account)",
+                    "podsecuritypolicies.v1beta1.policy (service accounts: system:kube-controller-manager)",
+                    "flowschemas.v1beta1.flowcontrol.apiserver.k8s.io (service accounts: system:serviceaccount:openshift-cluster-version:default)",
+                    " ".join([
+                        "prioritylevelconfigurations.v1beta1.flowcontrol.apiserver.k8s.io",
+                        "(service accounts: system:serviceaccount:openshift-cluster-version:default)",
+                    ])
                 ])
             }
 

